DEVELOP.rst

```diff
@@ -24,17 +24,33 @@ Then, run::
 Testing
 =======
 
-To test the theme, copy some docs into the ``docs`` directory.
+Mirror the documentation from `the CrateDB repository`_::
 
-Once your docs are in place, run this::
+    $ mkdir blackbox
+    $ cp -R ../crate/sql sql
+    $ cp -R ../crate/blackbox/docs blackbox/docs
 
-    $ sphinx-build -n -W -b html -E `pwd`/docs `pwd`/out/html
+These commands assume that your clone of the CrateDB repository is located at
+``../crate``, relative to your clone of the docs theme repository. If this
+isn't true, adjust the paths.
 
-If you want to build the documentation of CrateDB you will need to install
-the ``sphinx-csv-filter`` package upfront::
+Now, install the ``sphinx-csv-filter`` package::
 
     $ pip install sphinx-csv-filter
 
+To work around an issue with conflicting Python paths, copy the ``sphinx``
+module to the main source directory::
+
+    $ cp -R env/lib/python3.4/site-packages/crate/sphinx src/crate/sphinx
```

Comment:
the problem is, as far as I can tell, that under `src`, you have `crate.theme`. but then the theme wants to import `crate.sphinx` which lives under the `env` directory structure. and I think this confuses Python? having two `crate` dirs on the PYTHONPATH, each one contributing a different module. so my ugly bodge was just to copy over the module